PublicGitArchive/pga/cmd/cache.go

```diff
@@ -125,13 +125,13 @@ func matchHash(dest, source FileSystem, name string) bool {
 	return localHash == remoteHash
 }
 
-func getIndex(ctx context.Context) (io.ReadCloser, error) {
+func getIndex(ctx context.Context, datasetName string) (io.ReadCloser, error) {
```

Comment:
Let's have an enum here

```go
type DatasetKind int

const (
    DatasetSiva DatasetKind = iota
    DaatsetUast DatasetKind = iota
)
```

Reply:
Why ? I mean here we just need the name to join with the url/cache, and elsewhere we don't really need an index with the `Name` methode

Reply:
ah ok